projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2014 Alex Deryskyba (alex@codesnake.com)
```

Comment:
The problem is nobody knows who has written what (or maybe copied from fork xyz) - codesnake said he hasn't done it (so obviously no need to add a copyright for that) but we are also unsure how to handle it properly - due we are unsure we stay with the "old" licence instead of our own.